autoload/leaderf/python/leaderf/commandExpl.py

```diff
@@ -73,16 +75,20 @@ def _defineMaps(self):
     def _acceptSelection(self, *args, **kwargs):
         """"""
         cmd = args[0]
-        try:
-            lfCmd(cmd)
-            lfCmd("call histadd(':', '%s')" % escQuote(cmd))
-        except vim.error as e:
-            error = lfEncode(str(repr(e)))
-            if "E471" in error:
-                # Arguments mandatory
-                lfCmd("call feedkeys(':%s ')" % escQuote(cmd))
-            else:
-                lfPrintError(e)
+
+        if self._getExplorer()._run_immediately:
+            try:
+                lfCmd(cmd)
+                lfCmd("call histadd(':', '%s')" % escQuote(cmd))
+            except vim.error as e:
+                error = lfEncode(str(repr(e)))
+                if "E471" in error:
+                    # Arguments mandatory
+                    lfCmd("call feedkeys(':%s ')" % escQuote(cmd))
```

Comment:
I think it is better to use `feedkeys(':%s ', 'n')`, i.e., Do not remap keys.

Reply:
> Do not remap keys.

That looks better!
I fixed.

Reply:
could u also modify here？
https://github.com/Yggdroot/LeaderF/blob/614a2fe042d1df9d682114a48335fa5387a4ae36/autoload/leaderf/python/leaderf/historyExpl.py#L136

Reply:
OK.
I fixed.